docs/glues/herbs2gql.md

```diff
@@ -166,7 +166,58 @@ const [gql, resolver] = usecase2mutation(givenAUseCase, resolverFunc, options)
 
 ### Apollo Errors and Err
 
-Herbs2gql translates Herbs [Known Errors​](/docs/usecase/result#known-errors) to Apollo Errors as described in the documentation.
+`herbs2gql` deals with errors in the default resolver. It translates the usecase's errors into graphql errors. By default, it throws `UserInputError` for `invalid arguments` or `invalid entities` errors, `ApolloError` for `not found`, `already exists`, and `unknown` errors, and `ForbiddenError` for `authorization` errors and any other kind of errors.
```

Comment:
here a table is better to represent correlation, what do you think about it?

Reply:
Done!